ci-operator/config/openshift/boilerplate/openshift-boilerplate-master.yaml

```diff
@@ -12,6 +16,9 @@ images:
     ocp_builder_rhel-8-golang-1.22-openshift-4.17:
       as:
       - brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.22
+    ocp_builder_rhel-8-golang-1.23-openshift-4.19:
+      as:
+      - brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.23
```

Comment:
`brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.22` doesn't appear in the Dockerfile anymore. You can remove it and keep `rhel_8_1.23` only.

Reply:
@danilo-gemoli so maybe I can share my goal and you can tell me if this is possible.

- In https://github.com/openshift/boilerplate/pull/503 I am trying to test if Go 1.23 works as expected. I want to see CI pass on the branch.
- In this repo, I am trying to add the new image I need to run the test suite with an image that is Go 1.23.
- I want to keep Go 1.22 configured here so that I can easily revert/not merge should something not work, while allowing others to keep working on branches with Go 1.22 and tests on master still pass.

Based on what you are telling me this seems not possible?

Reply:
Ok, if that's the case you can have both and eventually keep tuning the Containerfile in your target repository.